src/main/java/Duke.java

```diff
@@ -1,10 +1,159 @@
+import java.util.Scanner;
+
+/**
+ * Chatbot program Duke that tracks user tasks.
+ *
+ * @author Leow Yuan Yang
+ * @version 4.0
+ * @since 2021-08-25
+ */
 public class Duke {
+    private static final String SEPARATOR = "\t==============================================";
+    private static final String logoArt = "\t                                        \n" +
+            "\t             ..........                 \n" +
+            "\t           .,,..,.,,,,                  \n" +
+            "\t           ,.,,,,.,...                  \n" +
+            "\t          ,.......,*,/& .               \n" +
+            "\t          ,..,..,((/**(((,              \n" +
+            "\t           *(#%(((#/#((/       *        \n" +
+            "\t         /(%#########%##(*,,,((*        \n" +
+            "\t              ((#(((((#      (((*       \n" +
+            "\t        ((((((   /((((/,., .... #       \n" +
+            "\t         (((#    .(((..*...,*///        \n" +
+            "\t         ((/      .,,**  ..**///        \n" +
+            "\t                  .,*,..  ,**//         \n" +
+            "\t                ..,,,..,/ ,*//          \n" +
+            "\t             *,,****,        " ;
+    private static final String nameArt = "\t _______  __   __  __   __  ______    _______ \n" +
+            "\t|       ||  |_|  ||  | |  ||    _ |  |       |\n" +
+            "\t|  _____||       ||  | |  ||   | ||  |    ___|\n" +
+            "\t| |_____ |       ||  |_|  ||   |_||_ |   |___ \n" +
+            "\t|_____  ||       ||       ||    __  ||    ___|\n" +
+            "\t _____| || ||_|| ||       ||   |  | ||   |    \n" +
+            "\t|_______||_|   |_||_______||___|  |_||___|    ";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        boolean isDone = false;
+        Task[] taskList = new Task[100];
```

Comment:
Perhaps this should be the plural of the object (eg : tasks), in accordance to the coding standards for this course.